cypresse2e/cypress/integration/e2eTests/signIn/signIn.spec.js

```diff
@@ -86,7 +86,7 @@ describe('FightPandemics Sign In Page', () => {
         });
 
         it('Login fail alert appears when incorrect email & password are entered and submitted by user', () => {
-            cy.generateRandomEmail().then((email) => {
+            cy.generateRandomEmail(8,8).then((email) => {
```

Comment:
We could create this 8 number as a constant somewhere else but this should be fine for now.